interop-tests/Cargo.toml

```diff
@@ -11,7 +11,7 @@ either = "1.8.0"
 env_logger = "0.9.0"
 futures = "0.3.26"
 if-addrs = "0.8.0"
-libp2p = { path = "../", default_features = false, features = ["websocket", "quic",  "mplex", "yamux", "tcp", "tokio", "ping", "noise", "tls", "dns", "rsa", "macros", "webrtc"] }
+libp2p = { version = "0.50.0", default_features = false, features = ["websocket", "quic",  "mplex", "yamux", "tcp", "tokio", "ping", "noise", "tls", "dns", "rsa", "macros", "webrtc"] }
```

Comment:
An alternative to this would be to branch off the `0.50.0` tag and copy the test in but I am not sure that buys us anything.